CHANGELOG.md

```diff
@@ -5,6 +5,10 @@
 #### v2.3.0
 
 - Bump to math.gl@3.3.0 (#968)
+- Gltf up axis support (#965)
```

Comment:
This is a new release `### 2.3.1`

Reply:
Not sure where the attributes loader came from, but it is OK.

Reply:
Changed to 2.3.1